ecs-agent/tmds/handlers/v4/handlers_test.go

```diff
@@ -288,6 +295,25 @@ func TestTaskMetadata(t *testing.T) {
 			expectedResponseBody: *expectedTaskResponse,
 		})
 	})
+
+	t.Run("happy case with FIS enabled", func(t *testing.T) {
```

Comment:
nit: Let's use "Fault Injection" instead of "FIS".

Reply:
Good catch! Will update.

Reply:
updated